README.md

```diff
@@ -48,3 +48,63 @@ Requests to cross-scheme subresources will no longer send SameSite=Lax or SameSi
+
+#### Scheme-Bound Cookies
+"Scheme-Bound Cookies" is concerend with the connection overwhich a cookie is being sent or set.
+In the current world a response from https://site.example/resource.jpg could set a cookie: `Set-Cookie: mycookie=value`.
+That cookie can then be sent to any url with "site.example" as the host regardless of scheme: http://site.example/otherresource.jpg, https://site.example/somethingelse.jpg, etc.
+
+With "Scheme-Bound Cookies" the browser will no longer send the cookie to a different scheme than the one that set it.
+With that in mind: a response from https://site.example/resource.jpg sets a cookie: `Set-Cookie: mycookie=value`. 
+That cookie now can **only** be sent to https://site.example urls. This includes cases where a different site, http://other.example, embedded resources from site.example: http://site.example/otherresource.jpg will **not** get sent the cookie, https://site.example/somethingelse.jpg will get sent the cookie.
+
+#### TL;DR
+"Schemeful Same-Site" affects the browsing context cookies will be sent/set in. "Is this url the same site as the one I'm on if I consider the schemes?"
+
+“Scheme-Bound Cookies” affects the connections the cookies will be sent on. "Does the scheme of the request url match the scheme of the original response url that set this cookie?"
+
+### Example
+This example showcases that it's possible to have a situation in which the cookies that "Schemeful Same-Site" and "Scheme-Bound Cookies" each send can differ.
+
+1. A user navigates to https://website.example which contains some embedded resources which want to set and read cookies.
+
+   * A response to the request for https://website.example/setsacookie.jpg has `Set-Cookie: mycookie=val; SameSite=Strict`
+
+2. The user then navigates to http://website.example which is the same page as before, served over an insecure connection.
+
+3. The browser checks to see if it can send the cookie to http://website.example/readsacookie.jpg
+
+   * In the current world (i.e. without either proposal) this cookie is allowed to be sent as this is a same-site context: only the registrable domain matters and they match.
+
+   * “Schemeful Same-Site” would also allow this cookie to be sent as we're still in a same-site context when considering scheme: the user is on http://website.example and the cookie wants to be sent to http://website.example
+
+   * “Scheme-Bound Cookies” would not allow this cookie to be sent as the connection isn't the same as the one the cookie was set by: the cookie was set on https and is trying to be sent on http.
+
+4. Immediately after the browser checks to see if it can send the cookie to https://website.example/readsacookie2.jpg
```

Comment:
"Immediately after" reads as if the browser dynamically retries the request for website.example/readsacookie2.jpg on an https connection. Is that what you mean? I don't think the mixed content work would apply here, since in step#2, the user is navigating to an http site?

Reply:
I was trying to make it clear that nothing interesting happens between these two steps. I think "Next the browser..." gets that point across without raising more questions.